client/components/mma/accountoverview/InAppPurchaseCard.tsx

```diff
@@ -16,13 +17,37 @@ const productTitleCss = css`
 	}
 `;
 
-export const InAppPurchaseCard = (props: {
+const summaryLinkCss = css`
+	color: currentColor;
+	text-decoration: underline;
+`;
+
+const cancelledAppSubscriptionMessage = (cancellationTimestamp: string) => {
+	return (
+		<>
+			Your app subscription was cancelled in{' '}
+			{dateString(new Date(cancellationTimestamp), 'MMMM yyyy')}. If you
+			would like to fund Guardian journalism again, please{' '}
+			<a css={summaryLinkCss} href="https://support.theguardian.com/">
+				support us today
+			</a>
+		</>
+	);
+};
+
+export const InAppPurchaseCard = ({
+	inAppPurchase,
+}: {
 	inAppPurchase: AppSubscription;
 }) => {
 	return (
 		<Stack space={3}>
-			{props.inAppPurchase.cancellationTimestamp && (
-				<InfoSummary message="ToDo: Your App Subscription Was cancelled" />
+			{inAppPurchase.cancellationTimestamp && (
+				<InfoSummary
+					message={cancelledAppSubscriptionMessage(
+						inAppPurchase.cancellationTimestamp,
+					)}
```

Comment:
You could use the `context` property for the supporting part of the message which would make things look a little nicer.

https://guardian.github.io/csnx/?path=/story/source-react-components-development-kitchen_info-summary--with-context